rkt/api_service_stub.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !sdjournal
+// +build !sdjournal,linux
```

Comment:
Actually no, now that I'm trying to remove it I realize that the whole api parts won't build on darwin, so it all is disabled. (If we don't disable it here it results in `rkt/api_service_stub.go:25: undefined: v1AlphaAPIServer`)